src/sparcscore/pipeline/workflows.py

```diff
@@ -725,14 +725,31 @@ def cellpose_segmentation(self, input_image):
             else:
                 continue
 
+        # check if there are any cytosol masks that are assigned to multiple nuclei
+        unique_pairs = list(set(nucleus_cytosol_pairs.values()))
+        # check if there are any duplicate values
+        if len(unique_pairs) != len(nucleus_cytosol_pairs):
+            # if there are duplicate values, set them to 0
+            seen_cytosol = set()
+            for nucleus_id, cytosol_id in nucleus_cytosol_pairs.items():
+                if cytosol_id in seen_cytosol:
```

Comment:
This only sets 0 for the second of the nucleus ids that are assigned to the same cytosol. We also want to get rid of the first one too though.